### DISCUSSION TITLE
Filters inside YOLOv5 architecture

### DISCUSSION THREAD

**Discussion by shubham-scisar** [ultralytics/yolov5#6572] [General]
Do the layers i.e. convolution layers use intrinsically any filters such as Edge detection,blur inside the architecture ?

**Comment by glenn-jocher** [comment]
@shubhamsharma1609 no. Deep learning models learn their own feature extraction convolution kernels, that's the whole point of deep learning in vision AI.

![image](https://user-images.githubusercontent.com/26833433/152988331-7b4471f8-e9a8-4193-a340-bf5e1ef9d262.png)

**Reply by shubham-scisar** [reply]
@glenn-jocher Apologies if my question was trivial. I agree with your explanation,but I was just trying to explore the architecture and  object detection through yolov5 (with internal working) and could not find any documentation related to the same.
